api/src/pcapi/core/finance/commands.py

```diff
@@ -124,3 +126,120 @@ def add_custom_offer_reimbursement_rule(
+    if source.siret != siret:
+        raise CheckError(f"Source venue {source.id} has SIRET {source.siret}, not requested SIRET {siret}")
+    if target.siret is not None:
+        raise CheckError(f"Target venue {target.id} already has a siret: {target.siret}")
+
+    if not override_revenue_check:
+        # Calculate yearly revenue of target venue
+        query = """
+          select sum(booking.amount * booking.quantity) as "chiffre d'affaires"
+          from booking
+          where
+            "venueId" = :target_id
+            and "dateUsed" is not null
+            -- On ajoute 1 heure pour la conversion UTC -> CET afin de récupérer
+            -- le chiffre d'affaires de l'année en cours.
+            and DATE_PART('year', "dateUsed" + interval '1 hour') = date_part('year', now() + interval '1 hour');
+        """
+        rows = db.session.execute(query, {"target_id": target_id}).fetchone()
+        revenue = rows[0]
+        if revenue and revenue > 10_000:
+            raise CheckError("Target venue has an unexpectedly high yearly revenue.")
+
+
+@blueprint.cli.command("move_siret")
+@click.option("--src-venue-id", type=int, required=True)
+@click.option("--dst-venue-id", type=int, required=True)
+@click.option("--siret", required=True)
+@click.option("--comment", required=True)
+@click.option("--apply-changes", is_flag=True, default=False, required=False)
+@click.option("--override-revenue-check", is_flag=True, default=False, required=False)
+def move_siret(
+    src_venue_id: int,
+    dst_venue_id: int,
+    siret: str,
+    comment: str,
+    apply_changes: bool = False,
+    override_revenue_check: bool = False,
+) -> None:
+    try:
+        check_can_move_siret(
+            source_id=src_venue_id,
+            target_id=dst_venue_id,
+            siret=siret,
+            override_revenue_check=override_revenue_check,
+        )
+    except CheckError as exc:
+        print(str(exc))
+        return
+
+    db.session.rollback()  # discard any previous transaction to start a fresh new one.
+    queries = [
+        """
+        update pricing
+        set "pricingPointId" = :target_id
+        where "pricingPointId" = :source_id
+        """,
+        """
+        update venue_pricing_point_link
+        set "pricingPointId" = :target_id
+        where "pricingPointId" = :source_id
+        """,
+        """
+        update venue
+        set siret = NULL, comment = :comment
+        where id = :source_id and siret = :siret
+        """,
+        """
+        update venue
+        set siret = :siret, comment = NULL
+        where id = :target_id and siret IS NULL
+        """,
+        """
+        insert into venue_pricing_point_link ("venueId", "pricingPointId", timespan)
+        values (
+          :target_id,
+          :target_id,
+          tsrange(now()::timestamp, NULL, '[)')
+        )
+        -- If the pricing of the target venue was the source venue before our changes,
```

Comment:
(détail) J'avais oublié des mots : 

```suggestion
        -- If the pricing point of the target venue was the source venue before our changes,
```